ckanext/canada/schemas/presets.yaml

```diff
@@ -3230,6 +3230,9 @@ presets:
       mimetype: application/x-filegdb
     - value: Flat raster binary
       replaces: [flat raster binary]
+    - value: GDB
+      mimetype: application/x-filegdb
+      label: GeoSoftDatabases
```

Comment:
We already have an entry for "FGDB/GDB"

Reply:
@PatLittle we have a bunch of these geospatial formats now that have unilingual labels that contain product names. Should we go back to using "file extension (+ description as required)" or is it ok to have unilingual labels for this controlled list?